libs/@blockprotocol/graph/src/stdlib/subgraph/element/entity.ts

```diff
@@ -54,80 +66,94 @@ export const getEntity = (
 
   const revisionVersions = Object.keys(entityRevisions).sort();
```

Comment:
Doesn't this only need to be sorted in the 'give me latest' case?

Reply:
Fixed as driveby in [Replace slice(-1) syntax](https://github.com/blockprotocol/blockprotocol/pull/928/commits/538ac50b82e2f9d62a01b9c9383f70e2536d0cc2)